tensilelite/Tensile/Components/LSU.py

```diff
@@ -245,6 +242,12 @@ def writeReadReduction(self, writer, kernel):
                 DSLoadBX     = DSLoadB32
                 numInstPerVW = bytesPerVector // 4
                 regsPerStore = 1
+            maxOffset =  (kernel["LocalSplitU"] -1) * ldsStride + ((numVgprPerLSU // self.LSUfullVw -1) * (numInstPerVW-1) + numInstPerVW) * regsPerStore * (bpr * kernel["WavefrontSize"])
```

Comment:
2 spaces before "(kernel["LocalSplitU..."
And the next line, 0 space before "maxOffset // ...."
We have no python code format checking so the code owner needs to take care by himself.